examples/contour/01-contour-config.yaml

```diff
@@ -99,3 +99,7 @@ data:
     #   Configure the number of additional ingress proxy hops from the
     #   right side of the x-forwarded-for HTTP header to trust.
     #   num-trusted-hops: 0
+    # rateLimitService:
+    #   extensionService: projectcontour/ratelimit
```

Comment:
We have comments for other bits in this config explaining what they mean, maybe match that a bit?

Reply:
added